muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/entries/user_credentials.ts

```diff
@@ -0,0 +1,9 @@
+import App from '~/containers/user_credentials';
+import reducer from '~/reducers/user_credentials';
+import runApp from '~/run';
+
```

Comment:
can we change the name to user-credentials.ts?

If we have been using that naming convention.